docs/development_suite/deploy/gitops-based/index.md

```diff
@@ -0,0 +1,35 @@
+---
+id: gitops-based
+title: GitOps deployment strategy
+sidebar_label: GitOps deployment strategy
+---
+
+:::caution
+This is a **BETA** feature and, as such, is currently under active development. Pay attention using it.
+:::
+
+**GitOps** is a modern approach to managing and automating the deployment of applications and infrastructure in a software development lifecycle. 
+By enabling the new **Enhanced Project Workflow** <!-- TODO: LINK -->on a Console project, it is possible to adopt a GitOps deployment strategy instead of the classic pipeline-based one.
```

Comment:
I'd add the info about the Enhanced workflow requirement in the above caution banner as well